src/components/Tetris/index.js

```diff
@@ -45,7 +45,7 @@ function Tetris(props) {
 			canvasFont: '17px "Press Start 2P"',
 			canvasFontColor: DARK_GRAY,
 			ghostColor: [LIGHT_GRAY, 'transparent'],
-			gameOverColor: ['#fff', LIGHT_GRAY],
+			gameOverColor: [NOTDARK_GRAY],
```

Comment:
I thought gameOverColor needed 2 elements in the array. What happens if you skip the second element? Let's add "transparent" if we don't need to set the color for the border